src/dmd/cli.d

```diff
@@ -241,6 +241,7 @@ struct Usage
         Option("cov",
             "do code coverage analysis"
         ),
+        Option("ctfe-cov", "Include code executed during CTFE in coverage report"),
```

Comment:
All coverage options should be part of the `-cov` switch. In this case, `-cov=ctfe` would seem best.

Reply:
so that would mean cov has be specified multiple time multiple times with different options.
If you want the percentage limit.
Which I found confusing.
e.g. `-cov=75, -cov=ctfe`

Reply:
I found @WalterBright 's suggestion to make more sense.
Also the discussion was about disabling it. I think it should be enabled by default, and disabled on demand.

Reply:
still it complicates command-line parsing, is there another CLI switch that takes arguments of different types?

Reply:
`-debug=<identifier>` and `-debug=<number>`?

Reply:
So what do people think about `-cov=no_ctfe` ?

Reply:
`-cov=ctfe` should probably be the way to go.

Reply:
Is that a "This will get in, when switches look that way"-statement? 
Or just an opinion of someone who is not going to merge it?
In that case I'd like @WalterBright opinion.

Reply:
`-cov=ctfe` to enable it, otherwise it is off.

Reply:
Done.
that's the case right now.